larq_compute_engine/__init__.py

```diff
@@ -1,3 +1,11 @@
-from larq_compute_engine.mlir.python.converter import convert_keras_model
+from larq_compute_engine.mlir.python.converter import (
+    concrete_function_from_keras_model,
+    convert_keras_model,
+    convert_tensorflow_graph,
+)
 
-__all__ = ["convert_keras_model"]
+__all__ = [
+    "convert_keras_model",
+    "concrete_function_from_keras_model",
+    "convert_tensorflow_graph",
```

Comment:
I would be in favour of not exposing this as a public API for now. For example the [`TFLiteConverter`](https://www.tensorflow.org/api_docs/python/tf/lite/TFLiteConverter) in TensorFlow 2.0 only supports `from_concrete_functions`, `from_keras_model` and `from_saved_model`. Which I think it would be good to adhere to that.

I am fine having this as an implementation detail, but I don't think it should be a public API. You could still import it using `larq_compute_engine.mlir.python.converter. convert_tensorflow_graph` if you really need to do some advanced stuff.

Reply:
Okay, in that case I don't think it makes sense to merge this at all. I tried doing some custom graph stuff but ended up needing to change `convert_tensorflow_graph` anyway, so I think if users want to convert a custom graph rather than a model, they will probably write their own function to do so.